locales-src/onboarding.ftl

```diff
@@ -19,6 +19,12 @@ onboarding-welcome-header = Welcome to { -brand-short-name }
 onboarding-welcome-body = You’ve got the browser.<br/>Meet the rest of { -brand-product-name }.
 onboarding-welcome-learn-more = Learn more about the benefits.
 
+onboarding-welcome-modal-variant-a-body = You’ve got the browser.<br/>Now get the most out of { -brand-product-name }.
+onboarding-welcome-modal-variant-b-body = Supercharge your browser's privacy protection.
```

Comment:
Please fix the apostrophes, it's a mix of ' and ’ right now.

Reply:
Good catch, done. Thanks!